localization/english/wc_game_rules_l_english.yml

```diff
@@ -0,0 +1,6 @@
+﻿l_english:
+ rule_wc_lore_characters_generate_families:0 "#Bold Warcraft#!: Generate Lore Characters Families"
+ setting_wc_lore_characters_generate_families_on:0 "On"
+ setting_wc_lore_characters_generate_families_on_desc:0 "If vanilla $rule_generate_families$ game rule is enabled, lore characters will be affected by it."
+ setting_wc_lore_characters_generate_families_off:0 "#high Off#!"
```

Comment:
I thought that the default option shouldn't be high.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/108640299-4a087100-74b2-11eb-8547-9efa9c6ac046.png)
![image](https://user-images.githubusercontent.com/46384992/108640300-4c6acb00-74b2-11eb-93ca-4ab82eb6807d.png)

</details>